packages/js/src/components/contentAnalysis/KeywordInput.js

```diff
@@ -123,24 +131,31 @@ KeywordInput.propTypes = {
 	isSEMrushIntegrationActive: PropTypes.bool,
 	displayNoKeyphraseMessage: PropTypes.bool,
 	displayNoKeyphrasForTrackingMessage: PropTypes.bool,
+	helpLink: PropTypes.string,
 };
 
 KeywordInput.defaultProps = {
 	keyword: "",
 	isSEMrushIntegrationActive: false,
 	displayNoKeyphraseMessage: false,
 	displayNoKeyphrasForTrackingMessage: false,
+	helpLink: "",
 };
 
 export { KeywordInput };
 
 export default compose( [
 	withSelect( ( select ) => {
-		const { getFocusKeyphrase, getSEMrushNoKeyphraseMessage, hasWincherNoKeyphrase } = select( "yoast-seo/editor" );
+		const { getFocusKeyphrase,
+			getSEMrushNoKeyphraseMessage,
+			getIsSEMrushIntegrationActive,
+			hasWincherNoKeyphrase } = select( "yoast-seo/editor" );
 		return {
 			keyword: getFocusKeyphrase(),
 			displayNoKeyphraseMessage: getSEMrushNoKeyphraseMessage(),
 			displayNoKeyphrasForTrackingMessage: hasWincherNoKeyphrase(),
+			isSEMrushIntegrationActive: getIsSEMrushIntegrationActive(),
+			helpLink: wpseoAdminL10n[ "shortlinks.focus_keyword_info" ],
```

Comment:
For the future, this is a way for those pesky global variables to not infect our components themselves.